Sources/PackageModel/InstalledSwiftPMConfiguration.swift

```diff
@@ -0,0 +1,30 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift open source project
+//
+// Copyright (c) 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+public struct InstalledSwiftPMConfiguration: Codable {
+    public struct Version: Codable, CustomStringConvertible {
+        let major: Int
+        let minor: Int
+        let patch: Int
+
+        public var description: String {
+            return "\(major).\(minor).\(patch)"
+        }
```

Comment:
I think we should also support `prerelease`, e.g. `-swift-5.9-DEVELOPMENT-SNAPSHOT-2023-04-25-b`